app/helpers/products_helper.rb

```diff
@@ -106,4 +106,12 @@ def with_c3_task(task)
     end
     [task]
   end
+
+  def provider_npi(provider)
```

Comment:
Actually, I just looked in HDS and it looks like it's in the Provider model already

https://github.com/projectcypress/health-data-standards/blob/master/lib/health-data-standards/models/provider.rb